en/mozillavpn.xlf

```diff
@@ -0,0 +1,505 @@
+    <trans-unit id="checkConnection">
+      <source xml:space="preserve">Check Connection</source>
+      <context-group purpose="location"><context context-type="linenumber">54</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/components/VPNControllerDevice.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="myDevices">
+      <source xml:space="preserve">My devices</source>
+      <context-group purpose="location"><context context-type="linenumber">22</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/views/ViewDevices.qml</context><context context-type="linenumber">44</context></context-group>
+    </trans-unit>
+    <trans-unit id="activeVsMaxDeviceCount">
+      <source xml:space="preserve">%1 of %2</source>
+      <context-group purpose="location"><context context-type="linenumber">45</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">53</context></context-group>
+      <note annotates="source" from="developer">Context: You have "x of y" (y is max) devices in your account.</note>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/components/VPNControllerServer.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="selectLocation">
+      <source xml:space="preserve">Select location</source>
+      <context-group purpose="location"><context context-type="linenumber">25</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/views/ViewServers.qml</context><context context-type="linenumber">17</context></context-group>
+      <note annotates="source" from="developer">Select the Location of the VPN server</note>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/components/VPNControllerView.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.deactivated">
+      <source xml:space="preserve">VPN is off</source>
+      <context-group purpose="location"><context context-type="linenumber">42</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">80</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">284</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.activationSloagan">
+      <source xml:space="preserve">Turn on to protect your privacy</source>
+      <context-group purpose="location"><context context-type="linenumber">48</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">85</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">290</context></context-group>
+    </trans-unit>
+    <trans-unit id="connecting">
+      <source xml:space="preserve">Connecting&#8230;</source>
```

Comment:
We should have the actual ellipsis instead of the entity.

The same applies to other entities across the file.